doc/nrf/gs_recommended_versions.rst

```diff
@@ -0,0 +1,23 @@
+.. _gs_recommended_versions:
+
+Required Tools
+##############
+
+The following table show the list of tools required for working with |NCS| **v1.3.0**.
+It lists the minimum required version as well as the version that is installed when installing using the :ref:`gs_app_tcm` as described in :ref:`nRF Connect for Desktop <gs_assistant>`.
```

Comment:
```suggestion
It lists the minimum required version and the version that is installed when using the :ref:`gs_app_tcm` as described in :ref:`gs_assistant`.
```

Reply:
Should we point out that this is the version that is used for automatic testing as well? @shantha-14

Reply:
Language comments addressed.